pom.xml

```diff
@@ -115,6 +115,9 @@
     <module>server</module>
     <module>client</module>
     <module>integration-test/common</module>
+    <module>metrics</module>
+    <module>metrics/metrics-core</module>
+    <module>metrics/metrics-prometheus</module>
```

Comment:
Why do we need another module? Could we add this code to common module?

Reply:
There will be more reporter in the future. Metric Reporters are optional and they may introduce many third party packages. Flink does the same thing.

Reply:
I think we need one module is enough. We can have different package `core` and `prometheus`.